Demos/DatAgora_PartDieu/ud-viz-context/config.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Can be removed since we now use the template (config.njk) to generate this file on the fly.